playbooks/manage-users/manage-local-user-access.yml

```diff
@@ -1,7 +1,7 @@
 ---
 
 - name: 'Update local access for account:{{ user_name }}'
-  hosts: all
+  hosts: update_access
```

Comment:
another option to provide a bit of safety-net would be `hosts: "{{ target }}"`  and pass it on the CLI

Reply:
@day4skiing No - I would like to keep this `all` ... this is a nice little playbook to update the password, and it should be possible to apply this to any inventory - hence the need for `all`